src/app/docs/examples/documentation/page.mdx

```diff
@@ -0,0 +1,196 @@
+docker push <your docker repository> .
+```
+
+## Deploy the MCP server
+
+Create a Secret for the OpenAI API Key which is going to be used to create the embeddings:
+
+```bash
+kubectl create secret generic mcp-secrets \
+  --from-literal=OPENAI_API_KEY=<your_openai_api_key>
+```
+
+Create a ConfigMap for the Database Configuration:
+
+```bash
+kubectl create configmap mcp-config \
+  --from-literal=SQLITE_DB_DIR=/data \
+  --from-literal=PORT=3001
+```
+
+Create a file named `deployment.yaml`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mcp-sqlite-vec
+  labels:
+    app: mcp-sqlite-vec
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mcp-sqlite-vec
+  template:
+    metadata:
+      labels:
+        app: mcp-sqlite-vec
+    spec:
+      containers:
+      - name: mcp-sqlite-vec
+        image: <your docker repository>
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 3001
+        env:
+        - name: OPENAI_API_KEY
+          valueFrom:
+            secretKeyRef:
+              name: mcp-secrets
+              key: OPENAI_API_KEY
+        - name: SQLITE_DB_DIR
+          valueFrom:
+            configMapKeyRef:
+              name: mcp-config
+              key: SQLITE_DB_DIR
+        - name: PORT
+          valueFrom:
+            configMapKeyRef:
+              name: mcp-config
+              key: PORT
+```
+
+Apply it:
+```bash
+kubectl apply -f deployment.yaml
+```
+
+Create a file named `service.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: mcp-sqlite-vec
+spec:
+  selector:
+    app: mcp-sqlite-vec
+  ports:
+  - port: 3001
+    targetPort: 3001
+  type: ClusterIP
+```
+
+Apply it:
+```bash
+kubectl apply -f service.yaml
+```
+
+## Add the MCP server in kagent
+
+In the kagent UI, click on `tools` and then on `Manage tool servers`.
+
+![manage tool servers](/images/manage-tool-servers.png "Manage tool servers")
+
+Click on `Add Server`.
+
+Call it `sqlite-vec`.
+
+Select `URL` and use `http://mcp-sqlite-vec.default:3001/sse`.
+
+![add tool server](/images/add-tool-server.png "Add tool server")
+
+Click on `Add Server`.
+
+After refreshing the page, you should see the `query-documentation` tool being discovered.
+
+![list tool servers](/images/list-tool-servers.png "List tool servers")
+
+## Create the MCP agent
+
+Click on `New Agent`
+
+Use the following information:
+
+- Agent Name: mcp-agent
+- Description: The MCP agent is answering questions about MCP, using the MCP documentation
+- Agent Instructions: Use your tool to answer any question about the Model Context Protocol (MCP).  Use `mcp` for the product and `latest` for the version.
+
+Click on `Add Tools`.
+
+Select the `query-documentation` tool.
+
+![list agents](/images/list-agents.png "List agents")
+
+Congratulations, the agent is ready !
+
+## Use the MCP agent
+
+Click on the agent and ask a question.
+
+For example, `How can a MCP server using sse ?`
```

Comment:
did you mean "How can you build an MCP server using sse?"

Reply:
Yes

Reply:
Fixed it